---
id:             "2022-1_yin_improving"
title:          "Improving the prediction of DNA-protein binding by integrating multi-scale dense convolutional network with fault-tolerant coding"
date:           2022-08-29 00:01:00 +0800
selected:       false
abbr:           'ABio 2023'
label:          ' <span class="badge badge-pill badge-custom badge-primary">SCI Q2</span> <span class="badge badge-pill badge-custom badge-primary">EI-Indexed Journal</span> '
pub_pre:        ' <span class="badge badge-pill badge-custom badge-secondary">Journal</span> '
pub:            "Analytical Biochemistry"
pub_last:       ''
pub_date:       "2022"

abstract: >-
  Accurate prediction of DNA-protein binding (DPB) is of great biological significance for studying the regulatory mechanism of gene expression. In recent years, with the rapid development of deep learning techniques, advanced deep neural networks have been introduced into the field and shown to significantly improve the prediction performance of DPB. However, these methods are primarily based on the DNA sequences measured by the ChIP-seq technology, failing to consider the possible partial variations of the motif sequences and errors of the sequencing technology itself. To address this, we propose a novel computational method, termed MSDenseNet, which combines a new fault-tolerant coding (FTC) scheme with the dense connectional deep neural networks. Three important factors can be attributed to the success of MSDenseNet: First, MSDenseNet utilizes a powerful feature representation approach, which transforms the raw DNA sequence into fusion coding using the fault-tolerant feature sequence; Second, in terms of network structure, MSDenseNet uses a multi-scale convolution within the dense layer and the multi-scale convolution preceding the dense block. This is shown to be able to significantly improve the network performance and accelerate the network convergence speed, and third, building upon the advanced deep neural network, MSDenseNet is capable of effectively mining the hidden complex relationship between the internal attributes of fusion sequence features to enhance the prediction of DPB. Benchmarking experiments on 690 ChIP-seq datasets show that MSDenseNet achieves an average AUC of 0.933 and outperforms the state-of-the-art method. The source code of MSDenseNet is available at https://github.com/csbio-njust-edu/msdensenet. The results show that MSDenseNet can effectively predict DPB. We anticipate that MSDenseNet will be exploited as a powerful tool to facilitate a more exhaustive understanding of DNAbinding proteins and help toward their functional characterization.

cover:          assets/images/covers/2022-1_yin_improving.jpg
authors:
  - Yu-Hang Yin
  - Long-Chen Shen
  - Yuan-Hao Jiang
  - Shang Gao
  - Dong-Jun Yu
PDF:          assets/PDF/2022-1_yin_improving.pdf
HTML:         https://doi.org/10.1016/j.ab.2022.114878
CODE_OR_DATASET:   https://github.com/csbio-njust-edu/msdensenet
BIB:
  "@article{2022-1_yin_improving,	</br>	title = {Improving the prediction of {DNA}-protein binding by integrating multi-scale dense convolutional network with fault-tolerant coding},	</br>	volume = {656},	</br>	issn = {0003-2697},	</br>	doi = {doi.org/10.1016/j.ab.2022.114878},	</br>	journal = {Analytical Biochemistry},	</br>	author = {Yin, Yu-Hang and Shen, Long-Chen and Jiang, Yuan-Hao and Gao, Shang and Song, Jiangning and Yu, Dong-Jun},	</br>	year = {2022},	</br>	pages = {114878},	</br>}"
---
